lib/ui/src/containers/nav.js

```diff
@@ -95,6 +97,41 @@ const createMenu = memoize(1)((api, shortcutKeys, isFullscreen, showPanel, showN
   },
 ]);
 
+export const removeLeafStories = stories => {
+  return Object.values(stories).reduce((acc, item) => {
+    // filter out leaves
+    if (item.isLeaf) {
+      return acc;
+    }
+
+    const { children, ...rest } = item;
+    const component = { ...rest, isLeaf: true, isComponent: true, id: children[0] };
+    // make components leaves
+    if (item.isComponent) {
+      return Object.assign(acc, { [item.id]: component });
```

Comment:
Is it weird that we end up with an entry in the `acc` who's key (`item.id`) is different to the `id` (`[children[0]`)?